testing_and_setup/seaice/configurations/standard_physics/streams.seaice

```diff
@@ -9,6 +9,12 @@
                   filename_interval="none"
                   input_interval="initial_only" />
 
+<immutable_stream name="SnicarInput"
```

Comment:
This isnt needed in this configuration. Please remove it.

Reply:
Removed.